kimai2-cmd.js

```diff
@@ -939,6 +944,11 @@ program.command('start [project] [activity]')
         const selected = {}
         checkSettings()
             .then(settings => {
+                if (!project || !activity) {
```

Comment:
```suggestion
                if (!project) {
```

I think this is enough, as you can't have an activity added without a project.

Reply:
Sorry, I'm a bit tired. It's ok.